src/utils/index.ts

```diff
@@ -2,3 +2,6 @@ export * from './extract-document-string-from-code-file';
 export * from './get-directives';
 export * from './get-implementing-types';
 export * from './validate-documents';
+export * from './get-schema-directive-from-directive-resolver';
+export * from './resolvers-composition';
+export * from './helpers';
```

Comment:
I'm not sure we need to export `helpers` because these are just internal method we use